### PR TITLE
[#3557] Implement LoRa Actility triangulated device location support

### PR DESCRIPTION
**Pull request by harism**
Implement support for device triangulated location in Hono LoRa adapter for Actility. The added location information is described in the Thingpark Wireless documentation; https://docs.thingpark.com/thingpark-wireless/6.1/Content/Resources/DocLibrary/TP-Wireless-6.0-LRC-OSS%20Interface%20Developer%20Guide(LoRaWAN).pdf (page 15)

The same documentation should apply also to Thingpark Enterprise specification and optional fields DevLAT, DevLON and DevAlt should be present in that adapter too.

### CODE REVIEW

**Comment by harism** [comment]
I didn't notice to sign ECA before making this pull request - should I recreate it from scratch or is it possible to continue with this same request now after I did sign ECA?

**Comment by calohmn** [comment]
@harism Thanks for the PR.

@BobClaerhout Do you have any comments on this?

**Comment by BobClaerhout** [comment]
> @harism Thanks for the PR.
> 
> @BobClaerhout Do you have any comments on this?

no remarks from me. LGTM

File: adapters/lora/src/main/java/org/eclipse/hono/adapter/lora/providers/ActilityEnterpriseProvider.java
Comment (calohmn) [inline adapters/lora/src/main/java/org/eclipse/hono/adapter/lora/providers/ActilityEnterpriseProvider.java:1]: Just some formalities:
Could you update the copyright year in this file (to `2020, 2023`), and in the other changed files?

And, could you create an issue for this feature request and reference it in the commit message?
Also, can you add your self/company to the list of copyright holders in `legal/src/main/resources/legal/NOTICE.md`?

**Comment by harism** [comment]
@calohmn I created an issue and tried to link its id to my commit and this PR, also I changed those years as you requested.

But the copyright holders list modification, I noticed that there is an open question about those here; https://github.com/eclipse-hono/hono/pull/3554#discussion_r1341819647 -- and I am wondering is this totally fundamental step or not?

**Comment by sophokles73** [comment]
> and I am wondering is this totally fundamental step or not?

I will to discuss this with the Eclipse Foundation but for the time being, it is ...

**Review by sophokles73 — APPROVED** [review]
LGTM

Thanks for contributing :+1:

**Comment by harism** [comment]
I modified the legal/NOTICE.md now too.

**Pull request merged**